ko/sql/jsp.rst

```diff
@@ -305,7 +342,179 @@ Java 저장 함수/프로시저에서 데이터베이스에 접근하기 위해
 
     Connection conn = cubrid.jdbc.driver.CUBRIDDriver.getDefaultConnection();
 
-서버 측 JDBC Driver에서 위와 같은 방법으로 데이터베이스에 연결하면 Java 저장 함수/프로시저 내에 존재하는 트랜잭션 관련 사항이 무시된다. 즉, Java 저장 함수/프로시저에서 수행되는 데이터베이스 연산은 Java 저장 함수/프로시저를 호출한 트랜잭션에 포함된다는 것을 의미한다. 아래의 Athlete 클래스에서 conn.commit()은 무시된다.
+.. note::
+
+    서버 측 JDBC 드라이버는 이미 등록되어 있기 때문에 Class.forName("cubrid.jdbc.driver.CUBRIDDriver")\를 호출하지 않아도 된다
+
+.. _jsp-execute-statement:
+
+질의문 수행
+--------------
+
+자바 저장/프로시저를 구현할 때 자바 어플리케이션을 개발하는 것과 동일하게 다음의 JDBC 인터페이스를 이용하여 질의문을 수행할 수 있다.
+
+    *    **java.sql.Statement**
+    *    **java.sql.PreparedStatement**
+    *    **java.sql.CallableStatement**
+
+다음은 위의 클래스를 사용하여 수행할 수 있는 질의문이다.
+
+*   **DML (Data Manipulation Language)**: :doc:`/sql/query/index`
+*   **DDL (Data Definition Language)**: :doc:`/sql/schema/index`
+
+.. note::
+
+    질의를 수행할 때 생성하는 JDBC 객체는 하나의 SQL 구문만 포함해야한다
+    따라서 다음의 경우에 에러가 발생한다
+    
+    stmt = new Statement ("select * from t1;select * from t2;");
+
+다음의 구문에 해당하는 기능은 지원하지 않는다.
+
+*   **TCL (Transaction Control Language)**: :ref:`database-transaction`
+
+.. note::
+
+    *    **COMMIT**, **ROLLBACK** 구문에 해당하는 함수인 *commit()*, *rollback()*\은 무시된다.
+    *    **SAVEPOINT** 구문에 해당하는 함수는 지원하지 않는다.
+
+질의문 수행 예시
+^^^^^^^^^^^^^^^^^
+
+**결과셋을 반환하는 질의 수행과 질의 결과셋 처리**
```

Comment:
```suggestion
**\결과셋을 반환하는 질의 수행과 질의 결과셋 처리**
```

Reply:
I also checked and backslush is not needed.